collector/zfs.go

```diff
@@ -35,9 +35,25 @@ type zfsSubsystemName string
 
 const (
 	arc            = zfsSubsystemName("zfsArc")
```

Comment:
Killed 'em and removed all but one type (zfsSysctl was actually useful because it worked around the inability to extend non-local types like string). I *hope* I got enough for your liking!